apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -166,31 +170,38 @@ export default function UserThreadPanel({
     setLoading(false)
   }
 
- 
-   const shouldFetchThreads = useCallback(() => {
-    if (isOpenPopup) return false;
-
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: handleThreadsChange

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)